src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -134,7 +136,7 @@ public function load($data, $type = null): RouteCollection
                             'collection' => $operation['collection'],
                             'operationId' => $operationId,
                         ],
-                    ] + ($operation['defaults'] ?? []),
+                    ] + ($operation['defaults'] ?? []) + $stateless,
```

Comment:
```suggestion
                    ] + ($operation['defaults'] ?? []),
```

Revert this and add after line 130:

```php
                       '_stateless' => $operation['stateless'],
```